models/DataObject/Classificationstore.php

```diff
@@ -107,10 +108,14 @@ public function setItems($items)
     }
 
     /**
+     * @param bool $withInheritance
      * @return array
      */
-    public function getItems()
+    public function getItems($withInheritance = false)
```

Comment:
Those additional parameters on public functions are breaking changes. It has to be done in a non-BC-breaking way.